recipe/meta.yaml

```diff
@@ -1,29 +1,20 @@
-{% set version = "4.1.4" %}
+{% set version = "4.1.5" %}
 
 package:
   name: zeromq
   version: {{ version }}
 
 source:
   fn: zeromq-{{ version }}.tar.gz
-  url: http://download.zeromq.org/zeromq-{{ version }}.tar.gz
-  sha1: b632a4b6f8a14390dc17824e37ff7b10831ce2b4
-  patches:
-    # Patch to avoid failing tests on Mac. See issues for details.
-    # https://github.com/zeromq/libzmq/issues/1509
-    # https://github.com/zeromq/libzmq/issues/1878
-    # https://github.com/zeromq/libzmq/issues/1464
-    - Makefile.am.mac.patch    # [osx]
-    # Patch to avoid a failing test on Linux. See issue for details.
-    # https://github.com/zeromq/libzmq/issues/1462
-    - Makefile.am.linux.patch  # [linux]
+  url: https://github.com/zeromq/zeromq4-1/archive/v{{ version }}.tar.gz
+  sha256: 5a993dcc330e63610fd74a6701b83b020aa5aedf22a98fea01eac93c72ff0b78
 
 build:
-  skip: true      # [win]
-  number: 3
+  number: 0
```

Comment:
Take a look at this [wiki]( https://github.com/conda-forge/staged-recipes/wiki/VC-features ) for how to setup VC to build correctly.